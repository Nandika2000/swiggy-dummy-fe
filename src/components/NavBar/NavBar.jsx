import React from 'react';


function NavBar() {
  return (    
      <nav className="navbar-content">
      <h1 className="title">SWIGGY</h1>
      </nav>
  );
}

export default NavBar;