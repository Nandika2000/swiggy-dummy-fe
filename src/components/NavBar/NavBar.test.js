import { render, screen } from "@testing-library/react";
import React from "react";
import NavBar from "./NavBar";

describe('NavBar component', () => {
    it("Should match snapshot", () => {
      const { container } = render(<NavBar />);
      expect(container).toMatchSnapshot();
    });
    test('should display "SWIGGY', () => {
      render(<NavBar />);
      screen.getByText("SWIGGY");
    });
  });