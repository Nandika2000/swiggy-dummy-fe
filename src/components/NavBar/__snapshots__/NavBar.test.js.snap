// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component Should match snapshot 1`] = `
<div>
  <nav
    class="navbar-content"
  >
    <h1
      class="title"
    >
      SWIGGY
    </h1>
  </nav>
</div>
`;
